lib/polymer_interop_config.dart

```diff
@@ -0,0 +1,27 @@
+library polymer_interop.polymer_interop_config;
```

Comment:
Lets stick this file under `lib/src`, and export it from `lib/polymer_interop.dart`.